app/src/main/res/values/strings.xml

```diff
@@ -124,8 +122,8 @@
     <string name="attack">Attack</string>
     <string name="defense">Defense</string>
     <string name="stamina_hp">HP</string>
-    <string name="missing_inputs">Missing data. Please fill CP and HP inputs.</string>
-    <string name="ivtext_no_possibilities">There are no IV possible combinations for this input. Please double check the information.</string>
+    <string name="missing_inputs">CP and HP are required.</string>
+    <string name="ivtext_no_possibilities">There are no possible combinations. Make sure the values you have entered are correct.</string>
```

Comment:
BTW I could also merge as-is and revise this later if you don't have objections to the text I propose.

And sorry for not replying earlier, was just slightly in a hurry and unsure of the best text.